pywbem/tupleparse.py

```diff
@@ -762,7 +767,7 @@ def parse_localnamespacepath(self, tup_tree):
             <!ELEMENT LOCALNAMESPACEPATH (NAMESPACE+)>
         """
 
-        self.check_node(tup_tree, 'LOCALNAMESPACEPATH', [], [], ['NAMESPACE'])
+        self.check_node(tup_tree, 'LOCALNAMESPACEPATH', [], [], ('NAMESPACE',))
```

Comment:
The `[]` could also be changed to an empty tuple, i.e. `()`.

Reply:
Agreed and you fixed.